src/pages/individual/EditIndividualMetadata.jsx

```diff
@@ -104,17 +120,24 @@ export default function EditIndividualMetadata({
           );
         })}
 
-        {error && (
+        {showErrorAlert && (
           <CustomAlert severity="error" titleId="SUBMISSION_ERROR">
-            {error}
+            {formErrors.length > 0 &&
+              formErrors.map(formError => (
+                <Text variant="body2">{formError}</Text>
+              ))}
+            {patchIndividualError && (
```

Comment:
I find myself wondering whether it's possible to just add the patchIndividual error to the formErrors array.

Reply:
That seems like a lot of extra work for very little payoff, though.

Reply:
I think there is a distinction between a formError and a requestError. From my perspective it's ok to keep them separate!